###############################################################################
#
# Copyright 2009 Sun Microsystems, Inc.  All rights reserved.
# Use is subject to license terms.
#
# Permission is hereby granted, free of charge, to any person obtaining a
# copy of this software and associated documentation files (the
# "Software"), to deal in the Software without restriction, including
# without limitation the rights to use, copy, modify, merge, publish,
# distribute, and/or sell copies of the Software, and to permit persons
# to whom the Software is furnished to do so, provided that the above
# copyright notice(s) and this permission notice appear in all copies of
# the Software and that both the above copyright notice(s) and this
# permission notice appear in supporting documentation.
# 
# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
# OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT
# OF THIRD PARTY RIGHTS. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
# HOLDERS INCLUDED IN THIS NOTICE BE LIABLE FOR ANY CLAIM, OR ANY SPECIAL
# INDIRECT OR CONSEQUENTIAL DAMAGES, OR ANY DAMAGES WHATSOEVER RESULTING
# FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT,
# NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION
# WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
# 
# Except as contained in this notice, the name of a copyright holder
# shall not be used in advertising or otherwise to promote the sale, use
# or other dealings in this Software without prior written authorization
# of the copyright holder.
#

Add notes on SMF integration and change to reflect default startup via
inetd.

diff -urp -x '*~' -x '*.orig' xfs.man xfs.man
--- xfs.man	2009-06-17 18:32:46.000000000 -0700
+++ xfs.man	2009-06-18 14:33:54.591588000 -0700
@@ -68,10 +68,14 @@ xfs \- X font server
 .B xfs
 is the X Window System font server.
 It supplies fonts to X Window System display servers.
-The server is usually run by a system administrator, and started via
-.BR init (__adminmansuffix__)
-or
-.BR inetd (__adminmansuffix__).
+The server is usually run by 
+.BR inetd (1M),
+as described in the NOTES section below.
+Automatic starting can be enabled or disabled through the
+.BR fsadmin (1)
+or 
+.BR inetadm (1M)
+commands.
 Users may also wish to start private font servers for specific sets of
 fonts.
 .PP
@@ -361,6 +365,22 @@ causes
 .B xfs
 to reset, closing all active connections and re-reading the configuration
 file.
+.\" Solaris/OpenSolaris additions for SMF:
+.SH "NOTES"
+.PP
+The \fBxfs\fR service is managed by the service management facility, 
+\fBsmf\fR(5), under the service identifier:
+.PP
+.nf
+svc:/application/x11/xfs:default
+.fi
+.PP
+Administrative actions on this service, such as enabling, disabling, or 
+requesting restart, can be performed using \fBsvcadm\fR(1M)\&. Responsibility 
+for initiating and restarting this service is delegated to \fBinetd\fR(1M)\&.
+Use \fBinetadm\fR(1M) to make configuration changes and to view configuration 
+information for this service\&. The service\&'s status can be queried using 
+the \fBsvcs\fR(1) command\&.
 .SH BUGS
 Multiple catalogues should be supported.
 .SH "FUTURE DIRECTIONS"
@@ -382,8 +402,11 @@ Keith Packard, Massachusetts Institute o
 .BR X (__miscmansuffix__),
 .BR xfsinfo (__appmansuffix__),
 .BR fslsfonts (__appmansuffix__),
-.BR init (__adminmansuffix__),
 .BR inetd (__adminmansuffix__),
-.BR syslog (__libmansuffix__),
+.BR fsadmin (1),
+.BR inetadm (1M),
+.BR smf (5),
+.BR syslogd (1M),
+.BR syslog (3c),
 .IR "The X Font Service Protocol" ,
 .I Font Server Implementation Overview
